content/docs/pulumi-cloud/cloud-rest-api/_index.md

```diff
@@ -3251,16 +3251,12 @@ Kubernetes Pods should not be used directly. Instead, you may want to use a Depl
 
 ## Environments
 
-{{< notes >}}
-Pulumi ESC (Environments, Secrets, and Configuration) and its associated REST API endpoints are currently in public preview.
-{{< /notes >}}
```

Comment:
I think for this we're actually going to need to do something similar to https://www.pulumi.com/docs/pulumi-cloud/deployments/api/#migrating-from-the-preview-api and we'll also need to add the note about ESC GA and how the preview routes are being deprecated (but that should go out next Wednesday).

But the migration guide should be necessary coz I think we changed some of the payloads from the preview routes.